src/test/java/uk/gov/pay/api/it/GetPaymentITest.java

```diff
@@ -63,7 +63,78 @@
     public void getPaymentWithMetadata() {
         publicAuthMock.mapBearerTokenToAccountId(API_KEY, GATEWAY_ACCOUNT_ID);
 
+        connectorMock.respondWithChargeFound(CHARGE_TOKEN_ID, GATEWAY_ACCOUNT_ID,
```

Comment:
This is so much cleaner with the builder and easier to see exactly what you're setting up for the test 👍